src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs

```diff
@@ -65,6 +65,11 @@ internal void Initialize(Guid workspaceGuid, Guid nodeGuid, string propName, str
             editText.TextArea.TextEntering += OnTextAreaTextEntering;
             editText.TextArea.TextEntered += OnTextAreaTextEntered;
 
+            // Initialize editor with global settings for show/hide tabs and spaces
+            editText.Options = dynamoViewModel.TextOptions;
+            editText.Options.ShowTabs = dynamoViewModel.ShowTabsAndSpacesInScriptEditor;
+            editText.Options.ShowSpaces = dynamoViewModel.ShowTabsAndSpacesInScriptEditor;
```

Comment:
I'm confused by this. If we have `TextOptions` in our VM, why do we also need `ShowTabsAndSpacesInScriptEditor` as well? Can't `ShowTabs` and `ShowSpaces` come already assigned in `TextOptions` so that we can avoid exposing `ShowTabsAndSpacesInScriptEditor` too?
Also, should this be needed if we are using a binding for `Options`?